lib/index.js

```diff
@@ -2,9 +2,4 @@
 //
```

Comment:
How much of this comment is still appropriate?

Reply:
Pruned.